src/main/java/blackjack/view/InputView.java

```diff
@@ -0,0 +1,47 @@
+package blackjack.view;
+
+import blackjack.domain.user.Dealer;
+import blackjack.domain.user.Name;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class InputView {
+    private static final Scanner SCANNER = new Scanner(System.in);
+    private static final String ASK_PLAYERS_NAME_MSG = "게임에 참여할 사람의 이름을 입력하세요. (쉼표 기준으로 분리)";
+    private static final String DELIMITER = ",";
+    private static final String MORE_DRAW_MSG_FORMAT = "%s는 한장의 카드를 더 받겠습니까?(예는 y, 아니오는 n)%n";
+    private static final String YES = "y";
+    private static final String NO = "n";
+    private static final String ANSWER_MUST_BE_YES_OR_NO_ERROR_MSG_FORMAT = "%s 또는 %s 로 입력해주세요.";
+
+    public static List<String> askPlayersName() {
+        System.out.println(ASK_PLAYERS_NAME_MSG);
+        return trimNames(splitNames(SCANNER.nextLine()));
+    }
+
+    private static List<String> splitNames(String names) {
+        return Arrays.asList(names.split(DELIMITER));
+    }
+
+    private static List<String> trimNames(List<String> splitNames) {
+        return splitNames.stream().map(String::trim).collect(Collectors.toList());
+    }
+
+    public static Boolean askMoreDraw(Name userName) {
+        System.out.printf(MORE_DRAW_MSG_FORMAT, userName);
+        String yesOrNo = SCANNER.nextLine();
+        validateYesOrNo(yesOrNo);
+        return yesOrNo.equalsIgnoreCase(YES);
+    }
+
+    private static void validateYesOrNo(String yesOrNo) {
+        if (!yesOrNo.equalsIgnoreCase(YES) && !yesOrNo.equalsIgnoreCase(NO)) {
+            throw new IllegalArgumentException(String.format(ANSWER_MUST_BE_YES_OR_NO_ERROR_MSG_FORMAT, YES, NO));
+        }
+    }
```

Comment:
- 비교 로직에서는 항상 더 확실한(안전한) 쪽을 기준으로 비교하는 것이 NPE를 방지하는 방어 로직입니다.
- 부정 연산자 (`!`) 는 가독성을 해치고, 실수를 유발할 소지도 있기 때문에 반대의 경우를 체크하거나, 메서드에 부정의 의미를 담아 분리하는 것이 좋습니다. (아래 제안은 하나의 예시입니다)

```suggestion
    public static boolean askMoreDraw(Name userName) {
        System.out.printf(MORE_DRAW_MSG_FORMAT, userName);
        String yesOrNo = SCANNER.nextLine();
        if (isValidAnswer(yesOrNo)) {
            return yesOrNo.equalsIgnoreCase(YES);
        }
        throw new IllegalArgumentException(String.format(ANSWER_MUST_BE_YES_OR_NO_ERROR_MSG_FORMAT, YES, NO));
    }

    private static boolean isValidAnswer(String yesOrNo) {
        return YES.equalsIgnoreCase(yesOrNo) || NO.equalsIgnoreCase(yesOrNo);
    }

```

Reply:
오..! 저는 변수명이 할당된 지역변수 String에게 상수를 물어보는게 자연스럽다고 여겨와서 항상 그렇게 코딩을 하였는데, NPE를 방지할 수 있는 습관이라는 내용이 확 와닿았습니다. 앞으로는 해당 습관을 길러보겠습니다! 기왕 하는 김에 NPE를 방지하는 습관들에 대해서도 학습해 보아야 겠네요.

또 스터디를 하다가 리뷰어님과 마찬가지로 부정연산자가 가독성을 해친다고 여겨 지양한다는 크루를 보고 저도 User에서 isSameStatus, isNotStatus를 만들어서 활용해 보았는데, 훨씬 읽기 좋아지는 느낌을 받았습니다. 
수정하겠습니다!